app/resources/views/pages/edits.txt

```diff
@@ -0,0 +1,5 @@
+
+	
+		if ( isProfileChanged() ) {
```

Comment:
When I tested this, it looked like the original usage of the dialog was removed.  You fixed it showing when we don't want it to to but you also made it not show when we do want it to.  In other words, you made it never show when we still want it to show whenever the user makes a change to the profile.

There are a couple comments on an older issue, #349, about the dialog.

Test that it shows exactly when we want it to.  Here are more detailed checks:
- Go to the profile and immediately try to navigate to another page like the home page.  No dialog should appear.
- Upload a profile image and click to rotate it.  Verify that no dialog shows up interfering with this.
- Change the location search radius without clicking "Save Profile".  Click "Home".  Verify that a dialog tells you that you have unsaved changes.
- Change anything else that would enable the "Save Profile" button and try to navigate away.  A dialog should appear.